src/openrct2/peep/Peep.cpp

```diff
@@ -2915,8 +2916,10 @@ static bool peep_interact_with_shop(Peep* peep, int16_t x, int16_t y, TileElemen
             money16 money = 0;
             guest->SpendMoney(money, cost, ExpenditureType::ParkRideTickets);
         }
-        peep->DestinationX = (x & 0xFFE0) + 16;
-        peep->DestinationY = (y & 0xFFE0) + 16;
+
+        auto coordsCenter = coords.ToTileCentre();
```

Comment:
```suggestion
        auto coordsCentre = coords.ToTileCentre();
```

We use British English throughout.